helpers/ea_mm2uniqueVoxInd.m

```diff
@@ -10,6 +10,7 @@
 
 % Remove all outliers
 voxSub = voxSub(all(voxSub, 2) & all(voxSub<=refnii.dim, 2), :);
+voxSub(any(voxSub<0,2),:)=[];
```

Comment:
this is the issue we discussed last time?

Reply:
Yes! for VTAs generated with certain stim params, the outliers are not removed and therefore errors out later in the fiber filtering pipeline